src/parser/core/calculateEffectiveDamage.ts

```diff
@@ -12,6 +12,21 @@ import { DamageEvent } from 'parser/core/Events';
  * @return the amount of damage attributable on the given damage event from the given boost
  */
 export default function calculateEffectiveDamage(event: DamageEvent, increase: number): number {
+  const overKill = event.overkill || 0;
   const raw = (event.amount || 0) + (event.absorbed || 0);
```

Comment:
this should use `overkill` right? total damage is `amount + absorbed + overkill`